README.md

```diff
@@ -5,50 +5,18 @@ It implements the protocol described in section 3 of [this](https://arxiv.org/ab
 
 More information about the protocol in this [presentation](https://youtu.be/Jj8rd4WOEy0) and this [blog post](https://comit.network/blog/2020/10/06/monero-bitcoin).
 
-## Quick start - CLI
+## Quick start
```

Comment:
I think we should mention the ASB in the main README and also point to the documentation for it.